transports/postgresql/index.md

```diff
@@ -11,16 +11,17 @@ The PostgreSQL transport implements a message queuing mechanism on top of [Postg
-|Scale-out                  |Competing consumer |
-|Scripted Deployment        |SQL Scripts |
-|Installers                 |Optional |
-|Native integration         |Supported, see [SQL statements used by the transport](https://github.com/Particular/NServiceBus.SqlServer/blob/master/src/NServiceBus.Transport.PostgreSql/PostgreSqlConstants.cs) |
+| Feature                                            |                                                                                                                                                                                    |
+|:---------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Transactions                                       | None, ReceiveOnly, SendsAtomicWithReceive                                                                                                                                          |
+| Pub/Sub                                            | Native                                                                                                                                                                             |
+| Timeouts                                           | Native                                                                                                                                                                             |
+| Large message bodies                               | PostgreSQL can handle arbitrary message size  within available resources, very large messages via data bus                                                                         |
+| Scale-out                                          | Competing consumer                                                                                                                                                                 |
+| Scripted Deployment                                | SQL Scripts                                                                                                                                                                        |
+| Installers                                         | Optional                                                                                                                                                                           |
+| Native integration                                 | Supported, see [SQL statements used by the transport](https://github.com/Particular/NServiceBus.SqlServer/blob/master/src/NServiceBus.Transport.PostgreSql/PostgreSqlConstants.cs) |
+| [time-to-be-received](#time-to-be-received) (TTBR) | Storage reclaimed at most 5 minutes after expiration when receiving endpoint is running                                                                                            |
```

Comment:
Why 5 minutes? The PostgreSQL transport does not have periodic task that deletes these messages. The only way to remove expired messages is by receiving them.